controllers/clusterdevfileregistrieslist_controller_test.go

```diff
@@ -1,6 +1,6 @@
 //
 //
-// Copyright 2022-2023 Red Hat, Inc.
```

Comment:
Why change the copyright header?

Reply:
Other repositories we changed all the source headers with the header file changes, reducing the future work for others changing these files (https://github.com/devfile/devfile-web/pull/95, https://github.com/devfile/api/pull/1256). I can retract these changes if we don't want to address this here.

Reply:
If we kept the changes, I will include the summary in the PR description.

Reply:
That's fine, I forgot about the license changes.